src/main/resources/public/jsx/appmanagement/myapps.jsx.js

```diff
@@ -263,7 +270,7 @@ var Instance = React.createClass({
                 </div>
                 <div className="row authority-app-services-title">
                     <div className="col-sm-12">
-                        <h4>{t('services')}</h4>
+                        <h4>{ applicationInstanceStatus !== 'PENDING' ? t('services') : '' }</h4>
```

Comment:
So no. We can't have an empty heading (which has a semantic purpose) just for display purposes.